src/frontends/onnx/docs/tests.md

```diff
@@ -4,9 +4,126 @@
+You can filter tests using `--gtest_filter` flag like any other gtest-based tests.
+
+Example command:
+```
+<OV_REPO_DIR>/bin/intel64/<OV_BUILD_TYPE>/ov_onnx_frontend_tests --gtest_filter=*add*
+
+```
+
+
+## Pre-steps for all Python tests:
+1. Build OpenVINO with `-DENABLE_PYTHON=ON`, preferably in a `Python` virtual environment. In order to avoid problems with many Python interpreters installed on the host, it is worth to set also `-DPYTHON_EXECUTABLE=PYTHON_INTERPRETER_PATH` build option.
+Note: if you want to run the tests from installation directory location (like in the CI), you should add `-P cmake_install.cmake` and `-DCOMPONENT=tests` cmake build options and install OpenVINO (via `cmake --build . --target install`) as additional steps.
+2. Set-up `Python` paths via `source <OV_INSTALL_DIR>/setupvars.sh` (or `sh <INSTALL_DIR>\setupvars.bat` for `Windows`).
+3. Instal `Python` dependencies:
+```
+pip install -r <OV_REPO_DIR>/src/bindings/python/requirements.txt
+pip install -r <OV_REPO_DIR>/src/bindings/python/requirements_test.txt
+```
+
+
+## Python frontend tests
+Run ONNX Frontend Python tests using commands:
+
+a) for build layout run:
+```
+pytest <OV_REPO_DIR>/src/bindings/python/tests/test_frontend/test_frontend_onnx*
+```
+b) for install layout run:
+```
+pytest <OV_INSTALL_DIR>/tests/pyopenvino/tests/test_frontend/test_frontend_onnx*
+```
+
+
+### Python ONNX operators tests
+Run ONNX operators Python tests using commands:
+
+a) for build layout run:
+```
+pytest <OV_REPO_DIR>/src/bindings/python/tests/test_onnx \
+    --ignore=<OV_REPO_DIR>/src/bindings/python/tests/test_onnx/test_zoo_models.py \
+    --ignore=<OV_REPO_DIR>/src/bindings/python/tests/test_onnx/test_backend.py
+```
+b) for install layout run:
+```
+pytest <OV_INSTALL_DIR>/tests/pyopenvino/tests/test_onnx \
+    --ignore=<OV_INSTALL_DIR>/tests/pyopenvino/tests/test_onnx/test_zoo_models.py \
+    --ignore=<OV_INSTALL_DIR>/tests/pyopenvino/tests/test_onnx/test_backend.py
+```
+
+
+### Python tests to confirm operators compliance with the ONNX standard
+a) for build layout run:
+```
+pytest <OV_REPO_DIR>/src/bindings/python/tests/test_onnx/test_backend.py -sv -k 'not cuda'
+```
+b) for install layout run:
+```
+pytest <OV_INSTALL_DIR>/tests/pyopenvino/tests/test_onnx/test_backend.py -sv -k 'not cuda'
+```
+
+
+## Python OpenModelZoo tests
+Pre: Download [models](https://github.com/onnx/models) (the current OpenVINO ONNX Frontend version uses [d58213534f2a4d1c4b19ba62b3bb5f544353256e](https://github.com/onnx/models/commit/d58213534f2a4d1c4b19ba62b3bb5f544353256e) version) or use [model_zoo_preprocess.sh](../../../../src/bindings/python/tests/test_onnx/model_zoo_preprocess.sh) script. The second approach is preferable, because some test data requires preprocessing.
```

Comment:
```suggestion
Preparation: Download [models](https://github.com/onnx/models). The current OpenVINO ONNX Frontend uses [this version](https://github.com/onnx/models/commit/d58213534f2a4d1c4b19ba62b3bb5f544353256e) or [the model_zoo_preprocess.sh script](../../../../src/bindings/python/tests/test_onnx/model_zoo_preprocess.sh). The second approach is preferable, because some test data requires preprocessing.
```